### PR TITLE
Use nginx by default

### PR DESCRIPTION
**Pull request by iderdik**
Trying to eliminate the post-deploy 503s

### CODE REVIEW

File: v2/setup/capcom.sh [hunk 1]
```diff
@@ -21,8 +21,8 @@ etcdctl set /CP/CP_KV_TTL 10
 etcdctl set /CP/CP_LOG_LEVEL $CAPCOM_LOG_LEVEL
 etcdctl set /CP/CP_LOG_LOCATION $CAPCOM_LOG_LOCATION
 etcdctl set /CP/CP_PORT 2002
-etcdctl set /CP/CP_PROXY haproxy
-etcdctl set /CP/CP_PROXY_CONFIG_FILE /etc/haproxy/haproxy.cfg
+etcdctl set /CP/CP_PROXY nginx
+etcdctl set /CP/CP_PROXY_CONFIG_FILE /etc/nginx/nginx.conf
 etcdctl set /CP/CP_PROXY_ENABLED true
-etcdctl set /CP/CP_PROXY_RESTART_SCRIPT /restart_haproxy.sh
+etcdctl set /CP/CP_PROXY_RESTART_SCRIPT /restart_nginx.sh
```
Comment (chrishenry) [inline v2/setup/capcom.sh:27]: LGTM. I'm assuming these are `ENV` variables passed to the capcom container?

**Comment by chrishenry** [comment]
Other question is; what's the procedure for deploying this?

**Comment by iderdik** [comment]
Procedure is to run those 3 statements, then perform a rolling restart of capcom.

File: v2/setup/capcom.sh (hunk 1 above)
Comment (iderdik) [inline v2/setup/capcom.sh:27]: Correct.

**Comment by chr0n1x** [comment]
:+1:

**Pull request merged**